<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AI_draughtClass</class>
 <widget class="QMainWindow" name="AI_draughtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI_draught</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="_0_1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_0_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_0_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_0_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_1_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_1_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_1_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_1_0">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_2_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_2_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_2_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_2_1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_3_0">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/black_flag.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_3_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_3_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_3_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_4_1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_flag.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_4_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_4_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_4_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_5_0">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_5_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_5_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_5_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_6_1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_6_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_6_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_6_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_7_0">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_7_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_7_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_7_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/AI_draught/white_empty.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Draw">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>和棋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Surrender">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>认输</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Finish_move">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>完成走子</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Help">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="start_white"/>
    <addaction name="start_black"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="start_white">
   <property name="text">
    <string>开始（白方）</string>
   </property>
  </action>
  <action name="start_black">
   <property name="text">
    <string>开始（黑方）</string>
   </property>
  </action>
  <action name="repentance">
   <property name="text">
    <string>悔棋</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="surrender">
   <property name="text">
    <string>认输</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AI_draught.qrc"/>
 </resources>
 <connections/>
</ui>
